source/agora/common/BlockStorage.d

```diff
@@ -504,8 +502,7 @@ public class BlockStorage
         }
         catch (Exception ex)
         {
-            logError("saveIndex: %s",
-                () @trusted { return ex.message; }());
+            () @trusted { stderr.writeln("saveIndex: ", ex.message); }();
```

Comment:
I assume we are going to replace this eventually with non-vibe.d logging?

Reply:
Yes